src/main/java/Product.java

```diff
@@ -0,0 +1,10 @@
+public class Product {
+    String name;
+    float cost;
+
+    Product(){
```

Comment:
🍏 Чтобы нельзя было создать невалидный товар (без имени или цены), лучше сделать конструктор, принимающий необходимые значения. Вызывать создание товара лучше тогда, когда все значения для него известны. А сами поля name и cost лучше сделать приватными, чтобы никто снаружи не мог из поменять.
```suggestion
    Product(String name, float cost){
        this.name = name;
        this.cost = cost;
    }
```